src/bench/bench_bitcoin.cpp

```diff
@@ -46,7 +46,7 @@ main(int argc, char** argv)
     std::string error;
     if (!gArgs.ParseParameters(argc, argv, error)) {
         fprintf(stderr, "Error parsing command line arguments: %s\n", error.c_str());
-        return false;
+        return 1;
```

Comment:
`EXIT_FAILURE` would be more explicit.

Reply:
Is it? I'm not sure. I've always found those constants quite useless.
I prefer using the explicit status codes, it is more flexible if want to use other codes than just 0 and 1 in the future.